Kitodo/src/main/webapp/WEB-INF/resources/css/kitodo.css

```diff
@@ -1179,17 +1179,22 @@ form#metadata div label,
 }
 
 #newCommentDialog .ui-dialog-content {
-    padding: 32px 16px;
+    padding: var(--default-double-size);
 }
 
 #newCommentDialog .ui-dialog-titlebar {
     display: none;
 }
 
+#newCommentForm h3 {
+    padding: 0;
+}
+
 .new-comment-box {
     width: 100%;
     background-color: var(--trans-blue);
     margin-bottom: 32px;
+    height: calc(var(--default-double-size) * 3) !important;
```

Comment:
What's overwriting the height of the element so you have to use `!important`? At least `.new-comment-box` sounds distinct enough so that it should not be an issue.

Reply:
Using the ID instead of the class did the trick.